RNTester/RNTester/AppDelegate.m

```diff
@@ -28,17 +37,24 @@ - (BOOL)application:(__unused UIApplication *)application didFinishLaunchingWith
 {
   _bridge = [[RCTBridge alloc] initWithDelegate:self
                                   launchOptions:launchOptions];
+  
+  // FIXME: this is a hack, remove when surfaces start correctly
+  [[NSNotificationCenter defaultCenter] addObserver:self
+                                           selector:@selector(handleJavaScriptDidLoadNotification:)
+                                               name:RCTJavaScriptDidLoadNotification
+                                             object:_bridge];
+  
+  _surfacePresenter = [[RCTSurfacePresenter alloc] initWithBridge:_bridge config:nil];
+  _bridge.surfacePresenter = _surfacePresenter;
 
   // Appetizer.io params check
-  NSDictionary *initProps = nil;
+  NSDictionary *initProps = @{};
   NSString *_routeUri = [[NSUserDefaults standardUserDefaults] stringForKey:@"route"];
   if (_routeUri) {
     initProps = @{@"exampleFromAppetizeParams": [NSString stringWithFormat:@"rntester://example/%@Example", _routeUri]};
   }
 
-  RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:_bridge
-                                                   moduleName:@"RNTesterApp"
-                                            initialProperties:initProps];
+  UIView *rootView = [[RCTFabricSurfaceHostingProxyRootView alloc] initWithBridge:_bridge moduleName:@"RNTesterApp" initialProperties:initProps];
```

Comment:
This needs to be configurable - it's ok to include fabric build in RNTester, but it shouldn't default to use Fabric yet (given it's not complete). It should default to the plain old RCTRootView, then via a toggle (a .plist, a global C flag, or something from NSUserDefault + simple UI to toggle, etc.), it should allow activating Fabric.

Reply:
@fkgozali for sure- I don't think we should merge these "hacks", but things like the podspecs should be good to go.

Reply:
That said, feature flagging it is pretty easy I suppose.

Reply:
What is your opinion on using a different scheme?

Reply:
C Flag is easiest heh.

Reply:
Added a new flag called `FABRIC_ENABLED`.

Reply:
C #define'ed flag can work, that way the compiler/build system can influence it.